source/publishing_data/07_configuring_visualizations/01_configuring_table_view/table.rst

```diff
@@ -1,22 +1,46 @@
-Configuring the table view
-==========================
+Configuring the Table visualization
+===================================
 
-.. admonition:: Note
-   :class: note
+.. admonition:: Prerequisite
+   :class: important
 
-   The table view is always available on all non-empty datasets.
+   For the Table visualization to be available, the dataset must not be empty.
```

Comment:
I would say "For the Table visualization to be available, the dataset must contain at least one source" ?

Reply:
Not true for the API proxy 😬 
but I agree with remi, empty means no record but you can still configure table viz (for instance when you initialize a realtime dataset with the schema, but it's still empty)